### ISSUE TITLE
I would like more customizetion

### ISSUE THREAD

**Issue by NertheritePig** [Linesmerrill/police-cad#613]
**Describe the solution you'd like**
[_A clear and concise description of what you want to happen._]

- I am trying to make a community but i can change stuff like the drivers lince and other stuff like so

**Implementation Details**
[_Add any and all specific things you would like to see implemented. These can include things such as limiting the input for a input box to only include specific characters. (This can be as technical as you would like to make it, including code examples)_]

- make it so we can cutomize stuff

**Comment by Linesmerrill** [comment]
Yeah great suggestion! We have had this in our backlog for a while now. We should work to prioritize this.

**Comment by Linesmerrill** [comment]
Done, download our mobile app to fully customize your CAD!
